package.json

```diff
@@ -24,11 +24,18 @@
    },
    "homepage": "https://github.com/romppane/Bulletin_Board#readme",
    "dependencies": {
+      "@hapi/boom": "^7.4.2",
+      "@types/boom": "^7.2.1",
       "@types/express": "^4.17.0",
+      "@types/hapi__boom": "^7.4.0",
+      "@types/moment-timezone": "^0.5.12",
+      "@types/morgan": "^1.7.35",
```

Comment:
All type packages should be in `devDependencies`. When installing types you can use `-D` like this for example `npm i -D @types/boom`